src/hotspot/share/gc/z/c2/zBarrierSetC2.cpp

```diff
@@ -253,11 +254,51 @@ static const TypeFunc* clone_type() {
   return TypeFunc::make(domain, range);
 }
 
+#define XTOP LP64_ONLY(COMMA phase->top())
```

Comment:
I'm curious whether `XTOP` should be underfined at the end of the scope (here and in other places).

Reply:
Added Undef in all four files that has it defined.